pybloqs/block/table_formatters.py

```diff
@@ -457,7 +457,7 @@ class FmtAlignTable(TableFormatter):
     """Set table alignment on page. Possible alignment paramters: left, center, right."""
 
     def __init__(self, alignment, rows=None, columns=None, apply_to_header_and_index=True):
-        super(FmtAlignTable, self).__init__(rows, columns)
+        super(FmtAlignTable, self).__init__(rows, columns, apply_to_header_and_index)
```

Comment:
This will not make a difference, since it only uses  the _create_table_level_css hook. But passing on the parameter for consistency is fine.